MAUI/DataGrid/Column-types.md

```diff
@@ -0,0 +1,916 @@
+---
+layout: post
+title: Column Types in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Column Types support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.
```

Comment:
description: Learn all about Column Types support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more here.